client/analytics/report/categories/table.js

```diff
@@ -85,7 +86,17 @@ class CategoriesReportTable extends Component {
 					value: getCurrencyFormatDecimal( net_revenue ),
 				},
 				{
-					display: numberFormat( products_count ),
+					display: (
+						<Link
+							href={
+								'admin.php?page=wc-admin#/analytics/products?filter=single_category&categories=' +
+								category_id
+							}
+							type="wp-admin"
```

Comment:
`<Link>` has a `wc-admin` type to make links shorter:
```ES6
						<Link
							href={
								'products?filter=single_category&categories=' +
								category_id
							}
							type="wc-admin"
						>
```

Reply:
You'll want to use `getPersistedQuery` here to make sure things like date parameters persist. 

```js
				<Link
					href={ getNewPath( persistedQuery, 'categories', {
						filter: 'single_category',
						categories: category.id,
					} ) }
					type="wc-admin"
				>
```

Reply:
Awesome- thanks for both of these tips!